src/rest.cpp

```diff
@@ -107,6 +107,23 @@ static CTxMemPool* GetMemPool(const std::any& context, HTTPRequest* req)
     return node_context->mempool.get();
 }
 
+/**
+ * Get the node context chainstatemanager.
+ *
+ * @param[in]  req The HTTP request, whose status code will be set if node
+ *                 context chainstatemanager is not found.
+ * @returns        Pointer to the chainstatemanager or nullptr if none found.
+ */
+static ChainstateManager* GetChainman(const std::any& context, HTTPRequest* req)
+{
+    auto node_context = util::AnyPtr<NodeContext>(context);
+    if (!node_context) {
```

Comment:
In commit "rest: Add GetChainman function and use it" (a839078c20f4ac3d92e14af7b287f2f0fddbc61e)

It seems like this should say `if (!node_context || !node_context->chainman)` otherwise the http error won't be set in the null chainman case.